cedar-dafny/validation/types.dfy

```diff
@@ -114,7 +145,11 @@ module validation.types {
     AttrNotFound(Type,Attr) |
     UnknownEntities(set<EntityType>) |
     ExtensionErr(Expr) |
-    EmptyLUB
+    EmptyLUB |
```

Comment:
-> EmptyLub to be consistent?

Reply:
The existing code is not self-consistent on this naming. I might fix in a later PR.